Assets/Scripts/ScriptingSystem/FunctionBlocks/MovePlayerBlock.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MovePlayerBlock : FunctionBlock
+{
+    [SerializeField] Transform player = null;
```

Comment:
No need to set null as value, null is default value compiler will set this for you.